data/brands/shop/convenience.json

```diff
@@ -5507,27 +5507,39 @@
       "displayName": "ローソン・スリーエフ",
       "id": "lawsonthreef-652b3e",
       "locationSet": {"include": ["jp"]},
-      "matchNames": [
-        "lawson+スリーエフ",
-        "ﾛｰｿﾝ・スリーエフ"
-      ],
+      "matchNames": ["ﾛｰｿﾝ・スリーエフ"],
       "tags": {
-        "brand": "ローソン・スリーエフ",
-        "brand:en": "LAWSON･Three F",
-        "brand:ja": "ローソン・スリーエフ",
+        "brand": "LAWSON+スリーエフ",
+        "brand:en": "LAWSON+Three F",
+        "brand:ja": "ローソン+スリーエフ",
         "brand:wikidata": "Q24866804",
         "name": "ローソン・スリーエフ",
-        "name:en": "Lawson･Three F",
+        "name:en": "Lawson-Three F",
```

Comment:
[JA:Naming sample](https://wiki.openstreetmap.org/wiki/JA:Naming_sample#shop=convenience) doesn't seem to be wrong.
If the value is correct, please correct JA:Naming sample.

Reply:
Since it is a matter of notation, JA:Naming sample would not be particularly wrong.
But rather than using the - instead of the ・, I thought it should be matched what is on the official site.